views/update/review-update-statement.html

```diff
@@ -49,7 +49,22 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds-from-desktop">
 
-      <span class="govuk-caption-xl govuk-!-padding-bottom-1">{{ appData.entity_name }} - {{ appData.entity_number }}</span>
+      {% include "includes/list/errors.html" %}
+      
+      <span class="govuk-caption-xl govuk-!-padding-bottom-1">
+        {% if appData.entity_name %}
```

Comment:
Can we avoid these conditionals by change to always include entityName and entityNumber in the session so nver using appData.entity_name/number?

Reply:
The first time the page loads it uses appData, then when an error is triggered the validation middleware spreads appData out and entity name and number can only be accessed through the variables directly. Because our current implementation iterates through BOs and MOs, it means that code then breaks as the data is spread. I'd need more time with it as it involves more investigation, but not worth holding the ticket back for given how close we are to the wire. This patch fix I made will ensure the ticket gets out in this sprint. By the looks of it a previous ticket has had to do the same thing a number of months back. I don't like it either but Its a weird one which I'll address with a bug fix after we get this ticket over the line.

Reply:
Ok for a bug fix but something to revisit.